Dockerfile

```diff
@@ -1,55 +1,9 @@
-FROM alpine:3.16
+FROM cimg/python:3.11.1-node
```

Comment:
I don't think we currently support 3.11, do we?

Reply:
nope, but this is the latest image from circleci so might as well update to the lastest.  we don't need to test sceptre with python 3.11